api_tests/preprints/views/test_preprint_actions.py

```diff
@@ -6,12 +6,12 @@
 )
 from osf.utils import permissions as osf_permissions
 
-from api_tests.reviews.mixins.filter_mixins import ReviewActionFilterMixin
+from api_tests.users.views.test_user_actions import TestReviewActionFilters
 from api_tests.reviews.mixins.comment_settings import ReviewActionCommentSettingsMixin
 
 
 @pytest.mark.enable_quickfiles_creation
-class TestPreprintActionFilters(ReviewActionFilterMixin):
```

Comment:
looks like `ReviewActionFilterMixin` is not used anywhere anymore -- unless i'm missing something, `test_filter_actions` (which you put such care into) is never actually run.

i think it'd be cleaner (and easier to keep track of) to put it back so both `TestPreprintActionFilters` and `TestReviewActionFilters` use `ReviewActionFilterMixin` directly, rather than one extending the other.